front-end/src/app/shared/models/transaction-types/EARMARK_RECEIPT.model.ts

```diff
@@ -20,6 +20,7 @@ export class EARMARK_RECEIPT implements TransactionType {
     return SchATransaction.fromJSON({
       form_type: 'SA11AI',
       transaction_type_identifier: ScheduleATransactionTypes.EARMARK_RECEIPT,
+      aggregation_group: "General",
```

Comment:
If we make AggregationGroup an enum, we can reference it directly here (and aggregation_group will be typed)